cadence/contracts/PMStrategies.cdc

```diff
@@ -588,22 +635,56 @@ access(all) contract PMStrategies {
         univ3FactoryEVMAddress: String,
         univ3RouterEVMAddress: String,
         univ3QuoterEVMAddress: String,
-        yieldTokenEVMAddress: String,
-        swapFeeTier: UInt32,
+        syWflowYieldTokenEVMAddress: String,
+        syWflowSwapFeeTier: UInt32,
         tauUsdfYieldTokenEVMAddress: String,
         tauUsdfSwapFeeTier: UInt32
```

Comment:
I think you can remove these values and encode them into setup transactions in `setup_mainnet.sh`